components/toggleButton/toggleButton.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * ToggleButton component.
+ */
+class ToggleButton extends Component {
+  /**
+   * Initialize default state
+   */
+  state = {
+    activeItem: 0,
+  }
+
+  /**
+   * Handlers onClick event
+   *
+   * @method handleSelectItem
+   * @param {Object} event
+   * @private
+   */
+  handleSelectItem = (event) => {
+    event.stopPropagation();
+
+    const index = this.props.items.indexOf(event.target.textContent);
+    this.setState({ activeItem: index });
+    (typeof this.props.handleSelect === 'function') && this.props.handleSelect(index);
```

Comment:
I'd only check if is set:

```js
this.props.handleSelect(index);
```

Since, based on the PropTypes it shouldn't be anything but a function and it's even required. 

If it wasn't required, you could also define it, as a defaultProp being a no-op, which would then make this easier:

```js
class ToggleButton extends React.Component {
  static defaultProps = {
    handleSelect = () => {},
  }
  
  /* ... */
  handleSelectItem = (event) => {
    /* ... */
   this.props.handleSelect(index);
   }
}
```

Still, looks good as it is already :) 👍  nice work